metadata.rb

```diff
@@ -6,6 +6,8 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version '0.10.0'
 
+depends 'yum-elrepo'
```

Comment:
This will need to get called out in the readme under the requirements section.  See https://github.com/chef-cookbooks/activemq/blob/master/README.md#cookbooks